git_machete/docs.py

```diff
@@ -182,6 +200,40 @@
           <b>--c, --checkout-my-github-prs</b>     Checkout your open PRs into local branches.
           <b>-y, --yes</b>                         Don't ask for confirmation when deleting branches from git.
     """,
+    "config": f"""
+        <b>Config keys: </b>
+        - `machete.github.(remote,organization,repository)`:
+
+          When executing `git machete github <subcommand>` command, {textwrap.indent(github_config_keys, "          ").strip()}
+
+        - `machete.overrideForkPoint.<branch>.(to,whileDescendantOf)`
+
+          Executing `git machete fork-point --override-to=<revision> [<branch>]`, sets up a fork point override for <branch>.
```

Comment:
```suggestion
          Executing `git machete fork-point --override-to=<revision> [<branch>]` sets up a fork point override for <branch>.
```

No need to separate subject and verb (_podmiot od orzeczenia_) by comma in English